phases/phase-2/challenges/become-validator.md

```diff
@@ -0,0 +1,50 @@
+# Become a testnet validator
+:::tip Note  
+Upon completing this challenge, you will be rewarded **300 Desmos Tokens**. 
```

Comment:
Upon completing this challenge, you will be rewarded **300 Desmos Tokens**. 
->
You will earn **300 Desmos Tokens** upon completion of this challenge.